crates/uv/src/commands/project/add.rs

```diff
@@ -93,6 +94,12 @@ pub(crate) async fn add(
     printer: Printer,
     preview: PreviewMode,
 ) -> Result<ExitStatus> {
+    if bounds.is_some() && preview.is_disabled() {
+        warn_user_once!(
+            "The bounds option is in preview and its configuration may change in any future release."
```

Comment:
```suggestion
            "The bounds option is in preview and may change in any future release."
```

Reply:
I guess `This option is in preview and may change in any future release.` is used elsewhere. I don't love that messaging. We can revisit separately though.